src/shared/components/person/settings.tsx

```diff
@@ -987,6 +1084,27 @@ export class Settings extends Component<any, SettingsState> {
     });
   });
 
+  handleInstanceSearch = debounce(async (text: string) => {
```

Comment:
With the massive number of federated instances we have now, we really need to add instance searching to `/search`, as well as do paging for `getFederatedInstances` . Obvi not rn, but ideally would be using the search endpoint like the others.